_posts/2022-09-27-#138.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: pr
+date: 2022-09-27
+title: "FROST"
+authors: ["jesseposner"]
+pr: 138
+components: ["crypto"]
+host: ariard
+status: upcoming
+commit: 6c1cf4e
```

Comment:
We'll probably need to start doing what Bitcoin Core Reviews does and have a review fork/branch of the project we're reviewing at this commit to avoid it getting lost on PR rebase etc. It's fine for now, but just something I can work on to improve the history (maybe when we start doing transcripts too).